### PR TITLE
Clarendon Option

### PR DESCRIPTION
**Pull request by kacycrider**

### CODE REVIEW

File: app/config/tstvhq/adventures.json [hunk 1]
```diff
@@ -66,23 +66,58 @@
...
+                        {
+                            "text": "Clarendon",
+                            "nextRoom": ["clarendon"]
+                        },
+                        {
+                            "text": "24 Please",
+                            "nextRoom": ["24"]
+                        }
+                    ]
+                },
```
Comment (ohnoabigshark) [inline app/config/tstvhq/adventures.json:119]: remove

File: app/config/tstvhq/adventures.json [hunk 2]
```diff
@@ -66,23 +66,58 @@
...
+                    ]
```
Comment (ohnoabigshark) [inline app/config/tstvhq/adventures.json:120]: remove

File: app/config/tstvhq/adventures.json [hunk 3]
```diff
@@ -66,23 +66,58 @@
...
+                    "text": "After a while, the voices drift off into music. You feel as if whatever was going on is over. Suddenly there is a resounding 'clunk' of internal mechanisms in the wall.  It opens, revealing a door to your left.",
                     "buttons": [
                         {
                             "text": "Go through the left door",
                             "nextRoom": ["trapDoor"]
                         },
                         {
                             "text": "Look for another way out",
-                            "nextRoom": ["library"]
+                            "nextRoom": ["library"]},
```
Comment (ohnoabigshark) [inline app/config/tstvhq/adventures.json:83]: remove the }, at the end of the line

File: app/config/tstvhq/adventures.json (hunk 1 above)
Comment (ohnoabigshark) [reply app/config/tstvhq/adventures.json:119]: remove 119**

File: app/config/tstvhq/adventures.json (hunk 2 above)
Comment (ohnoabigshark) [reply app/config/tstvhq/adventures.json:120]: remove 120**

File: app/config/tstvhq/adventures.json [hunk 4]
```diff
@@ -66,23 +66,57 @@
...
+                        } 
+                    ]
+                },
+ {
```
Comment (ohnoabigshark) [inline app/config/tstvhq/adventures.json:87]:
```suggestion
 {
```

File: app/config/tstvhq/adventures.json (hunk 3 above)
Comment (kacycrider) [reply app/config/tstvhq/adventures.json]: it is done

File: app/config/tstvhq/adventures.json (hunk 4 above)
Comment (ohnoabigshark) [inline app/config/tstvhq/adventures.json:87]: remove lines 87, 88, and 89

File: app/config/tstvhq/adventures.json (hunk 2 above)
Comment (kacycrider) [reply app/config/tstvhq/adventures.json]: it is done

File: app/config/tstvhq/adventures.json (hunk 4 above)
Comment (kacycrider) [reply app/config/tstvhq/adventures.json]: it is done

File: app/config/tstvhq/adventures.json [hunk 5]
```diff
@@ -81,10 +81,45 @@
...
+                            "text": "Clarendon",
+                            "nextRoom": ["clarendon"]
+                        },
+                        {
+                            "text": "24 Please",
+                            "nextRoom": ["24"]
+                        }
+                    ]
+                }
+          }
```
Comment (hndrewaall) [inline app/config/tstvhq/adventures.json:123]: still have a mismatched brace. slightly non obvious due to indentation. i can help over chat/zoom

**Review by hndrewaall — CHANGES_REQUESTED** [review]

**Review by hndrewaall — APPROVED** [review]

**Pull request merged**